app/src/main/java/com/teamagam/gimelgimel/app/Alerts/view/BubbleAlertsSubcomponent.java

```diff
@@ -78,10 +79,10 @@ private void onToolbarClicked() {
         private boolean mIsAnimating;
 
 
-        private PeakColorToolbarAnimator(Toolbar toolbar, Resources resources) {
-            mToolbarOriginalColor = resources.getColor(R.color.colorPrimaryDark, null);
+        private PeakColorToolbarAnimator(Toolbar toolbar, Context context) {
+            mToolbarOriginalColor = ContextCompat.getColor(context, R.color.colorPrimaryDark);
```

Comment:
You're right, I had issues with getting it's color. 
I'll give it another shot.